pkg/deviceshifu/deviceshifutcp/deviceshifutcp.go

```diff
@@ -0,0 +1,130 @@
+package deviceshifutcp
+
+import (
+	"fmt"
+	"github.com/edgenesis/shifu/pkg/logger"
+	"io"
+	"net"
+
+	"github.com/edgenesis/shifu/pkg/deviceshifu/deviceshifubase"
+	"github.com/edgenesis/shifu/pkg/k8s/api/v1alpha1"
+)
+
+type DeviceShifu struct {
```

Comment:
DeviceShifuTCP here?

Reply:
Also considered consistency to other Deviceshifu, like opcua [here](https://github.com/Edgenesis/shifu/blob/60da51054a230fe6ae47436c0cceb96730484cdc/pkg/deviceshifu/deviceshifuopcua/deviceshifuopcua.go#L22).

Reply:
ok